applications/sink/xmpp-sink/pom.xml

```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
-            <version>1.16.1</version>
+            <version>1.17.1</version>
```

Comment:
This is a minor bump, I think we should stick to patch versions.

Reply:
Removed